client/test/unit/specs/components/InterestModal.spec.js

```diff
@@ -121,15 +121,15 @@ describe('Unit | Component | InterestModal.vue', () => {
         expect(component.$data.error).to.equal(null);
       });
 
-      it('should trackEvent', () => {
+      it('should trackEventInterestClick', () => {
         // given
-        sinon.stub(component, 'trackEvent');
+        sinon.stub(component, 'trackEventInterestClick');
 
         // when
         component.submitInterest();
 
         // then
-        expect(component.trackEvent).to.have.been.calledWith();
+        expect(component.trackEventInterestClick).to.have.been.calledWith();
```

Comment:
to.have.been.called devrait suffire.
calledWith sans arguments me rend triste.
Mais sinon tu peux donner les arguments d'appel, ça me rendra encore plus heureux. :)

Reply:
ceci est un stub, il n'y a pas d'argument => should trackEventInterestClick without arguments